torch/csrc/jit/operator_upgraders/README.md

```diff
@@ -0,0 +1,95 @@
+# Guidance for Operator Developer
```

Comment:
We should have an intro on what the base concepts are: operator upgrader, BC and BC breaking, ...
Keep it short and simple.
You may already have something in the design review.

Reply:
Added